content/en/docs/instrumentation/js/serverless.md

```diff
@@ -173,3 +173,123 @@ function.
+
+### Setup environment variable for otelwrapper 
+
+Add the environment variable `NODE_OPTIONS` with the following value:
+
+```shell
+--require ./otelwrapper.js
+```
+
+### Select runtime
+Select Node.js version 16.
+
+### Establish otel wrapper
+
+Create a new file called `otelwrapper.js`, that will be used to instrument your service. Please make sure that you provide a `SERVICE_NAME` and that you set the `<address for your backend>`
+
+```javascript
+/* otelwrapper.js */
+
+const { Resource } = require('@opentelemetry/resources');
+const { SemanticResourceAttributes } = require ('@opentelemetry/semantic-conventions');
+
+
+const api = require("@opentelemetry/api");
+const { BatchSpanProcessor } = require("@opentelemetry/sdk-trace-base");
+const {
+  OTLPTraceExporter,
+} = require("@opentelemetry/exporter-trace-otlp-http");
+const { NodeTracerProvider } = require("@opentelemetry/sdk-trace-node");
+const { registerInstrumentations } = require("@opentelemetry/instrumentation");
+const {
+  getNodeAutoInstrumentations,
+} = require("@opentelemetry/auto-instrumentations-node");
+
+const providerConfig = {
+  resource: new Resource({
+      [SemanticResourceAttributes.SERVICE_NAME]: "<your function name>",
+  }),
+};
+
+api.diag.setLogger(new api.DiagConsoleLogger(), api.DiagLogLevel.ALL);
+
+const provider = new NodeTracerProvider(providerConfig);
+const collectorOptions = {
+  url: "<address for your backend>",
+};
+
+const spanProcessor = new BatchSpanProcessor(
+  new OTLPTraceExporter(collectorOptions)
+);
+
+provider.addSpanProcessor(spanProcessor);
+provider.register();
+
+registerInstrumentations({
+  instrumentations: [
+    getNodeAutoInstrumentations(),
+  ],
+});
+```
+
+### Establish package.json
+Add following content to default package.json
+
+```json
+{
+  "dependencies": {
+    "@google-cloud/functions-framework": "^3.0.0",
+    "@opentelemetry/api": "^1.3.0",
+    "@opentelemetry/auto-instrumentations-node": "^0.35.0",
+    "@opentelemetry/exporter-trace-otlp-http": "^0.34.0",
+    "@opentelemetry/instrumentation": "^0.34.0",
+    "@opentelemetry/sdk-node": "^0.34.0",
+    "@opentelemetry/sdk-trace-base": "^1.8.0",
+    "@opentelemetry/sdk-trace-node": "^1.8.0",
+    "@opentelemetry/resources": "^1.8.0",
+    "@opentelemetry/semantic-conventions": "^1.8.0"
+  }
+}
+```
+
+### Add HTTP call to function
+The following code makes a call to the OpenTemetry web site to demonstrate an outbound call
+
+```javascript
+const functions = require('@google-cloud/functions-framework');
+const https = require('https');
+
+functions.http('helloHttp', (req, res) => {
+
+    let url = "https://opentelemetry.io/";
+    https.get(url, (response) => {
+           res.send(`Response ${response.body}!`);
+      }).on('error', (e) => {
+           res.send(`Error ${e}!`);
+    })
+});
```

Comment:
```suggestion
functions.http('helloHttp', (req, res) => {
  let url = "https://opentelemetry.io/";
  https.get(url, (response) => {
    res.send(`Response ${response.body}!`);
  }).on('error', (e) => {
    res.send(`Error ${e}!`);
  })
});
```

Just doing 2-space formatting here since that seems to be typical for JS (although I imagine in our docs we're not exactly uniform about this either)